bookwyrm/models/status.py

```diff
@@ -258,6 +258,22 @@ def pure_content(self):
     pure_type = 'Article'
 
 
+class ReviewRating(Review):
+    ''' a subtype of review that only contains a rating '''
+    def save(self, *args, **kwargs):
+        if not self.rating:
+            raise ValueError('Rating object must include a numerical rating')
+        return super().save(*args, **kwargs)
+
+    @property
+    def pure_content(self):
+        #pylint: disable=bad-string-format-type
+        return 'Rated "%s": %d stars' % (self.book.title, self.rating)
```

Comment:
oh that's good to know! I should really get on board with the new string formatting syntax. it wasn't registering as an int to the linter I think because it's a custom model field wrapping IntegerField.